src/stepDefinitions/sharingContext.js

```diff
@@ -286,10 +286,16 @@ Given(
 )
 
 Given('the administrator has enabled exclude groups from sharing', function() {
+    if (client.globals.ocis) {
```

Comment:
fix the indentation.

Reply:
did our linter not working

Reply:
was not configured for this. i've made a pr to catch this.

Reply:
> https://github.com/owncloud/owncloud-test-middleware/pull/125
it's merged, please rebase.